configure.ac

```diff
@@ -2229,7 +2229,9 @@ fi
 ### Check for KLU library and header.
 
 OCTAVE_CHECK_LIB(klu, KLU,
-  [KLU library not found.  This will result in some lack of functionality for sparse matrices.],
+   [KLU library not found.  This will result in some lack of functionality for sparse matrices.
```

Comment:
The indentation was presumably fixed by you. Did you do that on a different branch?

Reply:
No, I did make sure to do this in this same branch, in [commit #a972f39](https://github.com/sarrah-basta/octave/commit/a972f391f5f26de0e02c62600f4ce9b898d73096) which is the commit on 22 March in this same branch as mentioned. Here is the permalink of the code file directly https://github.com/sarrah-basta/octave/blob/a54ef63c08d3a709d441d94978b816d1e85fe1a5/configure.ac#L2231-L2236

Reply:
Yeah. But you merged this PR instead of keeping it open. Hence, making it very difficult to keep an overview of the changes...